flink-runtime/src/main/scala/org/apache/flink/runtime/ActorLogMessages.scala

```diff
@@ -30,13 +30,15 @@ trait ActorLogMessages {
     override def isDefinedAt(x: Any): Boolean = _receiveWithLogMessages.isDefinedAt(x)
 
     override def apply(x: Any):Unit = {
-      if(log.isDebugEnabled) {
-        log.debug(s"Received message ${x} from ${self.sender}.")
```

Comment:
The idea of this log message is to see which message are currently processed by the actor. In cases where an actor fails it might be helpful to know which message he currently processed.